packages/@storefront/core/src/services/autocomplete.ts

```diff
@@ -53,7 +53,9 @@ class AutocompleteService extends LazyService {
   }
 
   hasActiveSuggestion() {
-    return this.registeredAutocompleteTags.some((tag) => tag.isActive());
+    return this.registeredAutocompleteTags.some(
+      (tag) => tag.isActive() && tag.select(Selectors.config).autocomplete.hoverAutoFill
```

Comment:
`tag.select(Selectors.config).autocomplete.hoverAutoFill` should technically be the same for all iterations. Is there a way we can pull this out outside the `some`?